config/environment.rb

```diff
@@ -3,3 +3,7 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+ActionView::Base.field_error_proc = Proc.new do |html_tag, instance|
```

Comment:
Unused block argument - `instance`. If it's necessary, use `_` or `_instance` as an argument name to indicate that it won't be used.